airgun/entities/puppet_environment.py

```diff
@@ -3,7 +3,7 @@
 from airgun.entities.base import BaseEntity
 from airgun.navigation import NavigateStep
 from airgun.navigation import navigator
-from airgun.views.puppet_environment import ImportPuppetEnvironmentView
+from airgun.views.puppet_environment import ChangedPuppetEnvironmentView
```

Comment:
This name is hard to understand outside of context of this PR. Could you rename that to `PuppetEnvironmentImportView`?

So:
* `airgun/entities/puppet_environment.py` will define `ImportPuppetEnvironmentView` with `navigator.register` decorator
* `airgun/views/puppet_environment.py` will define `PuppetEnvironmentImportView`

This will be consistent with other class names around.